.github/workflows/update_packages.yml

```
@@ -39,6 +39,7 @@ jobs:
                 if ($tested) {
                   git add "packages/$package/"
                   $newVersion = $version
+                  $finalUpdateType = $UPDATE_TYPE
```

[nitpick] I think documenting why this is done would help understanding the code
```suggestion
                  # Save the update type to use it in the commit
                  $finalUpdateType = $UPDATE_TYPE
```